homeassistant/components/flume/sensor.py

```diff
@@ -140,6 +139,11 @@ def unique_id(self):
         """Device unique ID."""
         return self._device_id
 
+    @property
+    def device_state_attributes(self):
+        """Return Flume state attributes."""
+        return self._attributes
```

Comment:
Please avoid including `"current_interval"` in the attributes since it is already the sensor state.  We store these in the database so every state update will have duplicate data.

We should probably round all the values to 1 decimal places (2 if we really need the precision) as that is what is done in the flume app as it will reduce the size of the data we store and reduce number of state updates.